dev-infrastructure/modules/acr/cache.bicep

```diff
@@ -0,0 +1,152 @@
+@description('Globally unique name of the Azure Container Registry')
+param acrName string
+
+@description('Location of the registry.')
+param location string
+
+@description('Name of the KV that contains the pull secrets for ACR caching.')
+param keyVaultName string
+
+type repositoryCache = {
+  @description('Name for the cache rule and credential set')
+  ruleName: string
+
+  @description('Source repository URL (e.g., "quay.io/openshift-release-dev/*")')
+  sourceRepo: string
+
+  @description('Target repository name (e.g., "openshift-release-dev/*")')
+  targetRepo: string
+
+  @description('Key Vault secret identifier for the username')
+  userIdentifier: string
+
+  @description('Key Vault secret identifier for the password')
+  passwordIdentifier: string
+
+  @description('Login server for the repository')
+  loginServer: string
+}
+
+@description('Array of Quay repositories to cache with their configurations')
+param quayRepositoriesToCache repositoryCache[]
+
+type purgeJob = {
+  @description('Name of the purge job')
+  name: string
+
+  @description('Filter pattern for images to purge')
+  purgeFilter: string
+
+  @description('How long ago to purge images (e.g., "2d", "7d")')
+  purgeAfter: string
+
+  @description('Number of images to keep')
+  imagesToKeep: int
+}
+
+@description('Array of purge job configurations')
+param purgeJobs purgeJob[] = []
```

Comment:
Is this param still being passed now that you switched from purge -> retention?

Reply:
Purge jobs still exist but we don't use them. Not being passed